### DISCUSSION TITLE
what kind of single cell RNA-seq data should be used for integration?

### DISCUSSION THREAD

**Discussion by haibol2016** [GreenleafLab/ArchR#1152] [Questions / Documentation]
@rcorces, @jgranja24, and @archrdevs 
I am trying to integrate scRNA-seq and scATAC-seq data from the same tissue type using ArchR' function-addGeneIntegrationMatrix. For scRNA-seq, I have data from 3 batches, while for scATAC-seq, I have 5 batches. Should I use a seurat object including all 3 batches or only one of the three batches? If I use a seurat object including all 3 batches, how the batch effect in scRNA-seq will affect downstream analyses based on GeneIntegrationMatrix, such as correlation calculation-based coaccessibility analysis?

Thank you very much for your comment and suggestion.

**Comment by rcorces** [comment]
I dont think I can give you an answer as to how your batch effects will affect downstream analyses. The integration will find the best possible match between ATAC and RNA and you should assess how well that matching looks based on your data and knowledge of your system. My advice would be to use all the cells you have rather than limit to a single batch.
